src/main/java/org/scoverage/plugin/SCoveragePreCompileMojo.java

```diff
@@ -320,11 +319,10 @@ else if ( "2.13".equals( resolvedScalaVersion ) || resolvedScalaVersion.startsWi
                 addScalacArgs = addScalacArgs + PIPE + "-Yrangepos";
             }
 
-            String _scalacPlugins =
-                String.format( "%s:%s:%s", pluginArtifact.getGroupId(), pluginArtifact.getArtifactId(),
-                               pluginArtifact.getVersion() );
+            String _scalacPlugins = pluginArtifacts.stream()
+                    .map(x -> String.format("%s:%s:%s", x.getGroupId(), x.getArtifactId(),x.getVersion())).collect(Collectors.joining(" "));
 
-            arg = PLUGIN_OPTION + pluginArtifact.getFile().getAbsolutePath();
+            arg = PLUGIN_OPTION + pluginArtifacts.stream().map(x -> x.getFile().getAbsolutePath()).collect(Collectors.joining(":"));
```

Comment:
```suggestion
            arg = PLUGIN_OPTION + pluginArtifacts.stream().map(x -> x.getFile().getAbsolutePath()).collect(Collectors.joining(String.valueOf(java.io.File.pathSeparatorChar)));
```

Reply:
@samantmaharaj could you amend this change to the PR?

Reply:
I've pushed that change now.